extension/templates/options.html

```diff
@@ -98,15 +98,15 @@ <h3>Article View</h3> <p><i>Below are the list of articles you have come across
                 
             </div>
         </div> -->
-        <br>
-        <hr>
-        <div class="row">
+        <!-- <br>
```

Comment:
pls remove these multiple comments cluster, and add a single comment

Reply:
I have made the changes accordingly and made the commit.